test/Common/Helpers/TestCaseHelper.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Extensions.ObjectPool;
```

Comment:
What is using this? Wasn't familiar with it but I also don't see anything using it in my reading of the file.

Reply:
nothing, omnisharp kept thinking I wanted `ObjectPool` instead of `object` 🤷